app/src/sharedTest/java/org/oppia/app/utility/ClickActions.kt

```diff
@@ -0,0 +1,28 @@
+package org.oppia.app.utility
+
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.CoordinatesProvider
+import androidx.test.espresso.action.GeneralClickAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Tap
+
+fun clickAtXY(pctX: Float, pctY: Float): ViewAction {
```

Comment:
What is ``pct``? Maybe just use: ``clickX`` & ``clickY``?

Reply:
done